client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
Mostly that we only do this because of testing needs right? Not a blocker at all, was just a thought.